### PR TITLE
update so the correct sitemap template is being uses (with alternate languages)

### PR DESCRIPTION
**Pull request by guyvanbael**

### CODE REVIEW

**Comment by dhensby** [comment]
I've noticed that you've created this pull request from one of the "core branches" rather than a branch specifically intended for this pull request. This can cause problems if we ask you to target this PR at a different branch. No action is needed at the moment, this is for your information.

In the future please create a new branch for each individual pull request. See our [guide about contributing PRs](https://docs.silverstripe.org/en/4/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request)

File: src/Control/FluentSitemapController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+     * @package fluent
+     * @author Damian Mooyman <damian.mooyman@gmail.com>
+     */
+    class FluentSitemapController extends GoogleSitemapController
+    {
+        public function init()
+        {
+            parent::init();
+            // Reset any session locale to use the default locale as the standard 'base' locale
+            //Fluent::set_persist_locale(Fluent::default_locale(true));
```
Comment (tractorcow) [inline src/Control/FluentSitemapController.php]: This class is redundant isn't it? Just use the default google sitemap controller.

File: src/Extension/FluentDirectorExtension.php [hunk 2]
```diff
@@ -70,9 +72,9 @@ public function updateRules(&$rules)
 
         // @todo - Build SitemapController after GoogleSitemap module is migrated to ss4
         // If Google sitemap module is installed then replace default controller with custom controller
-        //if (class_exists(GoogleSitemapController::class)) {
-        //    $rules['sitemap.xml'] = SitemapController::class;
-        //}
+        if (class_exists(GoogleSitemapController::class)) {
+            $rules['sitemap.xml'] = FluentSitemapController::class;
```
Comment (tractorcow) [inline src/Extension/FluentDirectorExtension.php]: If you use a fluent-namespaced controller you need the fluent-namespaced template also.

Just use the default googlesitemapcontroller class from will.r's module.

Comment (guyvanbael) [reply src/Extension/FluentDirectorExtension.php]: If you use the the default Wilr's default googlesitemapcontroller, then the default GoogleSitemapController_sitemap.ss gets rendered and that one doesn't contain the alternate languages. Or is there a way to route the default controller to the FluentSitemapController.ss?

**Comment by tractorcow** [comment]
Replaced with https://github.com/tractorcow/silverstripe-fluent/pull/352

**Pull request closed without merging**